index.html

```diff
@@ -241,15 +286,51 @@ <h3>
       credential/request data=], which the user agent MAY match against a
       holder's software, such as a digital wallet.
     </p>
+    <h2>
+      Extensions to `CredentialCreationOptions` dictionary
+    </h2>
+    <pre class="idl">
+    partial dictionary CredentialCreationOptions {
+      DigitalCredentialCreationOptions digital;
+    };
+    </pre>
+    <h3>
+      The `digital` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="CredentialCreationOptions">digital</dfn> member
+      allows for options to configure the issuance of a [=digital credential=].
+    </p>
+    <h2>
+      The `DigitalCredentialCreationOptions` dictionary
+    </h2>
+    <pre class="idl">
+    dictionary DigitalCredentialCreationOptions {
+      sequence&lt;DigitalCredentialRequest&gt; requests;
```

Comment:
maybe: 
```suggestion
      required sequence&lt;DigitalCredentialRequest&gt; requests;
```

Based on #210

Reply:
This should be a `DigitalCredentialCreation` dictionary instead , which has the required request...

Reply:
I have decided to go for
`DigitalCredentialGetRequest` and `DigitalCredentialCreateRequest`